main.go

```diff
@@ -0,0 +1,36 @@
+package main
+
+import (
+	"github.com/fogo-sh/gutenbuild/runtime"
+
+	"github.com/BurntSushi/toml"
+
+	"os"
+
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+
+	"io/ioutil"
```

Comment:
Figured out what was odd about these imports - appears `goimports` treats blank spaces as something absolutely unchangable and refuses to reformat these imports. If you remove the blank lines and save `goimports` should reformat this into the format I expected (which I've got below as a suggestion for your easy application)

```suggestion
	"io/ioutil"
	"os"

	"github.com/BurntSushi/toml"
	"github.com/fogo-sh/gutenbuild/runtime"
	"github.com/rs/zerolog"
	"github.com/rs/zerolog/log"
```